README.md

```diff
@@ -1,31 +1,147 @@
-# kube-batch
+# Volcano
 
 [![Build Status](https://travis-ci.org/kubernetes-sigs/kube-batch.svg?branch=master)](https://travis-ci.org/kubernetes-sigs/kube-batch)
 [![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes-sigs/kube-batch)](https://goreportcard.com/report/github.com/kubernetes-sigs/kube-batch)
 [![RepoSize](https://img.shields.io/github/repo-size/kubernetes-sigs/kube-batch.svg)](https://github.com/kubernetes-sigs/kube-batch)
 [![Release](https://img.shields.io/github/release/kubernetes-sigs/kube-batch.svg)](https://github.com/kubernetes-sigs/kube-batch/releases)
 [![LICENSE](https://img.shields.io/github/license/kubernetes-sigs/kube-batch.svg)](https://github.com/kubernetes-sigs/kube-batch/blob/master/LICENSE)
 
+Volcano is system for runnning high performance workloads on
```

Comment:
It should specify right at the outset its a  batch scheduler, or a scheduler for long running jobs. 'sysem' is very overloaded word

Reply:
volcano is not only a scheduler, it's a batch "system" that include others features.